### PR TITLE
Earth 698 earth display

### PR DESCRIPTION
**Pull request by buttonwillowsix**
# NOT READY

# Summary
- Change the order of fields in the Earth Events Display and do re-styling as needed.

# Needed By (Date)
- End of sprint

# Urgency
- Medium

# Steps to Test

1. Go to an event node
2. Take alook to see that When, Where, etc. are up under the title
3. Less important info (audience, etc.) is below the description
4. Social media links are moved under the image instead of under the title/subtitle info

# Affected Projects or Products
- I added some general styling to social actions class since it is used multiple places. I think more consolidation is possible.

# Associated Issues and/or People
- EARTH 698

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by buttonwillowsix** [comment]
Got that irritating thing where changed images are pulled in along with everything

File: templates/stanford_event/stanford-event.html.twig [hunk 1]
```diff
@@ -53,6 +40,54 @@
...
   </div>
 
+  <div class="content-type__header">
+    <div class="page-title"><h1>{{ title }}</h1></div>
+    {% if sub_title is not empty %}
+    <div class="page-subtitle">
+      <h2>
+        {{ sub_title }}
+      </h2>
+      </div>
```
Comment (sherakama) [inline templates/stanford_event/stanford-event.html.twig]: syntax indenting is off by two spaces.

**Comment by sherakama** [comment]
@buttonwillowsix is this ready?

**Comment by buttonwillowsix** [comment]
I want to look at it one time with Barb before I call it done. I will fix syntax then too!

**Comment by buttonwillowsix** [comment]
@sherakama, Barb is fine with this display for now. So if the syntax is fixed, this is good to go.

**Review by sherakama — APPROVED** [review]

**Pull request merged**